client-sdk/ts-web/core/reflect-go/go.mod

```diff
@@ -1,12 +1,13 @@
 module github.com/oasisprotocol/oasis-sdk/client-sdk/ts-web/core/reflect-go
 
-go 1.24.0
+go 1.25.0
 
-toolchain go1.24.2
+replace github.com/cometbft/cometbft => github.com/oasisprotocol/cometbft v0.37.15-oasis2
 
-replace github.com/cometbft/cometbft => github.com/oasisprotocol/cometbft v0.37.15-oasis1
-
-require github.com/oasisprotocol/oasis-core/go v0.2503.0
+require (
+	github.com/oasisprotocol/oasis-core/go v0.2506.0
+	golang.org/x/tools v0.36.0
```

Comment:
Should this go in the other group? (doesn't seem related to aosis-core)

Reply:
I think not, as here are direct imports and below are indirect. And `go mod tidy` will throw `x-tools` under direct.